packages/app/components/Queue/QueueListSharedComponents.tsx

```diff
@@ -99,15 +98,35 @@ const QueueRoomGroup = styled.div`
 const DisableQueueButton = styled(QueueInfoColumnButton)`
   color: white;
   background: #da3236;
-  bottom: 0;
-  position: absolute;
   &:hover,
   &:focus {
-    color: white;
-    background: #f76c6c;
+    color: #da3236;
+    background: #fff;
+    border-color: #da3236;
   }
 `;
 
+const ClearQueueButton = styled(QueueInfoColumnButton)`
+  background-color: #ff8c00;
```

Comment:
look at this vibrancy of this orange 😎 
![image](https://user-images.githubusercontent.com/23366632/165009139-24e99420-f130-40de-8c0b-6ee96172532f.png)